src/parser/csql_grammar.y

```diff
@@ -27903,6 +27903,7 @@ pt_create_char_string_literal (PARSER_CONTEXT *parser, const PT_TYPE_ENUM char_t
         length = node->info.value.data_value.str->length;
 
         node->type_enum = char_type;
+/*
```

Comment:
In the query below, the tailing space of a varchar column is treated like a char column. Is your intention correct?
`select * from tbl where varchar_col = 'abc  '`

Reply:
yes, the policy of varchar-type constant is changed.

Reply:
It would be nice if you could write in a comment that you would like to comment on the policy change.

Reply:
Ok, I commented at JIRA as you said.